src/ws-dropdown/ws-dropdown.js

```diff
@@ -301,6 +303,10 @@ export class WSDropdown extends Component {
     item.classList.add(animationClass);
   }
 
+  disabledStyle() {
+    return this.props.disabled ? ' disabled' : '';
```

Comment:
We aligned on a naming convention which says that states are prefixed with an `is-` and modifies with an `mod-`
So could you please rename that class to an `is-disabled`?

As well it would be cool if you would create a variable inside the render function since this function is used only there and we don't need to blow up the function amount.